### PR TITLE
Extracting group and role information to use in policy evaluation

### PR DESCRIPTION
**Pull request by Bento007**

### CODE REVIEW

File: fusillade/clouddirectory.py [hunk 1]
```diff
@@ -1127,7 +1149,7 @@ class PolicyMixin:
 
     def lookup_policies(self) -> List[str]:
         policy_paths = self.cd.lookup_policy(self.object_ref)
-        return self.cd.get_policies(policy_paths)
+        return self.cd.get_policies(policy_paths)['policies']  # TODO use roles and groups extraced from policy
```
Comment (DailyDreaming) [inline fusillade/clouddirectory.py]: `extraced`

File: fusillade/clouddirectory.py [hunk 2]
```diff
@@ -1444,7 +1466,7 @@ def lookup_policies(self) -> List[str]:
             policy_paths = self.lookup_policies_batched()
         else:
             raise AuthorizationException(f"User {self.status}")
-        return self.cd.get_policies(policy_paths)
+        return self.cd.get_policies(policy_paths)['policies']  # TODO use roles and groups extraced from policy
```
Comment (DailyDreaming) [inline fusillade/clouddirectory.py]: `extraced`

File: fusillade/clouddirectory.py (hunk 1 above)
Comment (Bento007) [reply fusillade/clouddirectory.py]: will be resolved in the next PR

File: fusillade/clouddirectory.py (hunk 2 above)
Comment (Bento007) [reply fusillade/clouddirectory.py]: will be resolved in the next PR

**Pull request merged**